aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -401,6 +401,45 @@ Resources:
 
 <%   if cdn_enabled && app == 'Pegasus' -%>
 <%=    component 'www_redirect', app: app -%>
+  MarketingRouterLambda:
+    Type: AWS::Lambda::Function
+    Properties:
+      Description: 'Lambda for dynamically routing marketing traffic to different origins based on the URL path'
+      FunctionName: <%="#{rack_env.to_s}-MarketingRouter"%>%
+      Code: <%=zip_directory 'marketing-router'%>
```

If it's just one file, we can use `js`.

```suggestion
      Code: <%=js 'marketing-router'%>
```